### PR TITLE
resolves #133 add -b --basic mode to only output final image update readme

### LINKED ISSUE #133: add "--simple" or "--single" mode to CLI, so you just get the final image
This could be `--simple -s` or `--single -s` -- not sure, but we should have a commandline option that makes it work more like ImageMagick's `convert` -- abandoning for this mode the folder of sequential images and just outputting the final image. This, in concert with #130, will make it a really useful and familiar utility for those used to `convert`.

Bonus followup -- make a wrapper which emulates syntax in the style of `convert`!

### PR DESCRIPTION
**Pull request by tech4GT**
Signed-off-by: tech4GT <varun.gupta1798@gmail.com>

### CODE REVIEW

**Comment by tech4GT** [comment]
@jywarren please have a look

File: index.js [hunk 1]
```diff
@@ -57,6 +58,8 @@ sequencer.setUI({
 sequencer.loadImages(program.image,function(){
     console.warn('\x1b[33m%s\x1b[0m', "The execution will be async\nYou may not see the output for a few seconds or minutes")
 
+    if(program.basic) console.log("Basic mode is enabled")
```
Comment (jywarren) [inline index.js]: Can we add `, outputting only final image` and also on line 19 in the `-h` help command? Thanks, this is great!

**Comment by jywarren** [comment]
Left a couple requests but this is really cool! Thanks!

**Comment by tech4GT** [comment]
@jywarren thankyou😊
anything else?

**Comment by jywarren** [comment]
This looks great! Have you tried building it with `grunt build` for the distributed version? With that I think we'd be good to go!

**Comment by tech4GT** [comment]
@jywarren yes i have and as a matter of fact travis also runs grunt build so it's double sure😊

**Comment by jywarren** [comment]
Oh, awesome! Can you check in the updated `/dist/....` files? Thanks!

**Comment by tech4GT** [comment]
@jywarren done😊

**Comment by jywarren** [comment]
Awesome!!! 🎉

**Comment by ccpandhare** [comment]
This was cool!

**Pull request merged**